examples/quickstart.ipynb

```diff
@@ -31,10 +31,10 @@
    },
    "outputs": [],
    "source": [
-    "!pip install autollm -Uq\n",
-    "!pip install gradio -Uq\n",
-    "!pip install gitpython -Uq\n",
-    "!pip install nbconvert -Uq"
+    "!pip install autollm -Uqq\n",
```

Comment:
lets write all dependencies in one line as !pip install autollm gradio gitpython nbconvert -Uqq